src/api/index.js

```diff
@@ -343,6 +358,82 @@ export function getInstallProperties() {
   return get(uri);
 }
 
+
+export async function patchServiceAccount(
+  serviceAccountName,
+  namespace,
+  secretName
+) {
+  const uri = getKubeAPI('serviceaccounts', {
+    name: serviceAccountName,
+    namespace
+  });
+  const patch1 = await patchAddSecret(uri, secretName);
+  return patch1;
+}
+
+// Get list of service accounts where secrets patched to, returns service account in full
+export async function getSecretServiceAccountList(saList, secretName) {
+  const secretServiceAccountList = [];
+
+  if (saList.length > 0) {
+    saList.forEach(element => {
+      const saSecretList = element.secrets;
+      let secretFound = false;
+      saSecretList.forEach(element1 => {
+        if (element1.name === secretName) {
+          secretFound = true;
+        }
+      });
+      if (secretFound === true) {
+        secretServiceAccountList.push(element);
```

Comment:
Why not add the push in the above if and omit this?

Reply:
Done 😸